publish/publish-type.proj

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <UsingTask TaskName="DownloadFromAzure" AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.CloudTestTasks.dll" />
+
+  <Import Project="$(PackagesDir)/$(FeedTasksPackage.ToLower())/$(FeedTasksPackageVersion)/build/$(FeedTasksPackage).targets" />
+
+  <PropertyGroup>
+    <BuildDependsOn>
+      ValidateProperties;
+      DownloadFilesFromContainer;
+      SignSymbolPackages
+    </BuildDependsOn>
+
+    <BuildDependsOn Condition="$(PublishType.Contains('blob')">
```

Comment:
Do we want to make these more explicit than `contains`?  Should we check the full `$(PublishType)` value?

Reply:
The PublishType property is intended to use contains checks to configure different parts of publish, so this is the right check and is the same thing we are doing in the build definitions directly.